src/website/app/demos/Button/examples/primary.js

```diff
@@ -15,12 +15,26 @@
  */
 
 /* @flow */
+import { createStyledComponent } from '../../../../../utils';
 import Button from '../../../../../Button';
 
+const DemoLayout = createStyledComponent('div', {
+  '& > button': {
+    marginRight: '0.5rem'
+  }
+});
+
 export default {
   id: 'primary',
   title: 'Primary',
-  description: 'Use once per page, for the primary action.',
-  scope: { Button },
-  source: `<Button primary>Do Something</Button>`
+  description: `Primary Buttons are used once per container, usually as the main call to action for the page.
```

Comment:
Buttons or buttons?

Reply:
honestly I wasn't sure about this one. I was leaning towards `Button` since we're talking about the Mineral UI implementation of a primary, but since we're talking about general usage, it could definitely be the other way. @kylegach thoughts?